chapters/ch33.md

```diff
@@ -1,28 +1,28 @@
 > # 33. Why we compare to human-level performance
 
- Tại sao chúng ta so sánh với hiệu quả cấp con người?
+ # 33. Tại sao chúng ta so sánh với hiệu quả cấp con người?
 
 > Many machine learning systems aim to automate things that humans do well. Examples include image recognition, speech recognition, and email spam classification. Learning algorithms have also improved so much that we are now surpassing human-level performance on more and more of these tasks.
 
- Nhiều hệ thống machine learning hướng tới tự động hóa những thứ con người làm tốt. Ví dụ như nhận dạng hình ảnh, nhận dạng giọng nói, và phân loại thư rác. Các thuật toán học cũng đã được cải thiện rất nhiều, đến mức vượt qua hiệu quả cấp con ngườii trong ngày càng nhiều tác vụ.
```

Comment:
> Nhiều hệ thống machine learning hướng tới tự động hóa những thứ con người làm tốt. Ví dụ như nhận dạng hình ảnh, nhận dạng giọng nói, và phân loại thư rác. Các thuật toán học cũng đã được cải thiện rất nhiều, đến mức vượt qua hiệu quả cấp con ngườii trong ngày càng nhiều tác vụ.

Thay "machine learning" bằng "học máy".

Reply:
@phamdinhkhanh Hiện tại từ "machine learning" được giữ nguyên từ đầu cuốn sách. Nếu muốn thay đổi thì cần thay đổi toàn bộ. Tạm thời giữ nguyên "machine leanring" cho nhất quán với các chương khác. Sau đó mình có thể thay lại toàn bộ từ đầu trong một PR khác.